crates/holochain_types/src/app.rs

```diff
@@ -370,16 +370,8 @@ impl InstalledApp {
     }
 }
 
-impl automap::AutoMapped for InstalledApp {
-    type Key = InstalledAppId;
-
-    fn key(&self) -> &Self::Key {
-        &self.app.installed_app_id
-    }
-}
-
 /// A map from InstalledAppId -> InstalledApp
-pub type InstalledAppMap = automap::AutoHashMap<InstalledApp>;
+pub type InstalledAppMap = IndexMap<InstalledAppId, InstalledApp>;
```

Comment:
This is to ensure consistent ordering of the serialized response, to allow a test to assert that the response matches an expected value.